pcsx2/gui/ConsoleLogger.h

```diff
@@ -220,3 +223,29 @@ class ConsoleLogFrame : public wxFrame
 
 	void OnLoggingChanged();
 };
+
+template<typename ... Args>
+void OSDlog(ConsoleColors color, bool console, const std::string& format, Args ... args) {
+	if(!GSosdLog) return;
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-security"
```

Comment:
The pragmas here generate 30 unknown pragma warnings in VS2015 builds.

It seems that the warnings GCC generates are due to `args` being empty, so perhaps you should also create a version of `OSDlog` that doesn't use variadic templates? You can then call that version from the variadic template version with an already processed string.